Add torchao as a depdency (#2386)

jerryzh168 opened the pull request:
Summary:



att

Differential Revision: D54831762

kimishpatel commented:
Can we take this dep only for llama2? We dont need this to be installed for executorch generally. At least not yet.

kimishpatel reviewed (changes requested):
make it llama2 specific for now since we havent expanded usage beyond that yet

Jack-Khuu reviewed (approved):
Stamp to unblock, dittoing Kimish

Please move this to executorch/examples/models/llama2/install_requirements.sh if we don't need it for all of ET

jerryzh168 committed:
Add torchao as a depdency (#2387)

Summary:


bypass-github-export-checks
bypass-github-pytorch-ci-checks
bypass-github-executorch-ci-checks

att

Reviewed By: Jack-Khuu

Differential Revision: D54831762
